CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 4.5.0
```

Comment:
Perhaps `4.4.1` instead of `4.5.0` for the proposed version number?
Since no new resources are being added, timeout seems like a small enhancement.
 I'd also suggest adding `(Unreleased)` here.

Reply:
Done.